department-of-veterans-affairs/vets-website
# [codespaces] add ability to run cypress GUI

## Summary

Adds the [desktop-lite](https://github.com/devcontainers/features/tree/main/src/desktop-lite) feature and command to allow using `cypress open` when developing with codespaces.

### How to test

1. Create a codespace according to the [documentation](https://depo-platform-documentation.scrollhelp.site/developer-docs/using-github-codespaces), starting from this branch
1. Run the command `yarn cy:open-codespaces`
1. Visit http://127.0.0.1:6080/
   1. log in with the password `vscode`
   1. confirm that you can use the cypress GUI

## Screenshots

![image](https://github.com/department-of-veterans-affairs/vets-website/assets/101649/5f4f4d6f-d477-4007-9215-dabf2ed0b145)

## What areas of the site does it impact?

- Codespaces config only
